consensus/qbft-core/src/main/java/org/hyperledger/besu/consensus/qbft/core/types/QbftFinalState.java

```diff
@@ -104,4 +104,25 @@ public interface QbftFinalState {
    * @return true if the local node is the proposer for the given round, false otherwise
    */
   boolean isLocalNodeProposerForRound(ConsensusRoundIdentifier roundIdentifier);
+
+  /**
+   * Gets the frequent round change multicaster.
+   *
+   * @return the frequent round change multicaster
+   */
+  public ValidatorMulticaster getFrequentRCMulticaster();
```

Comment:
This is misnamed as it resending more than just round changes

```suggestion
  public ValidatorMulticaster getFrequentMulticaster();
```